CMakeLists.txt

```diff
@@ -86,6 +86,9 @@ option(WarpX_amrex_internal                    "Download & build AMReX" ON)
 # change the default build type to RelWithDebInfo (or Release) instead of Debug
 set_default_build_type("RelWithDebInfo")
 
+# Option to enable interprocedural optimization
+option(WarpX_IPO                                "Compile with interprocedural optimization (will take more time)" OFF)
```

Comment:
Please document the new option also in our `cmake.rst` build docs :)

Reply:
Ah, sure!

Reply:
It should be OK now.  I chose "interprocedural optimization" because that is the name used by `cmake`. However, if it is preferable, I can change it to LTO...

Reply:
Sounds great, yes I was basically wondering if we want to add the aliases just in case, e.g. in brackets. Will take a look at the line in the docs, thanks!